src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
```

Comment:
As I understand you use this one font everywhere in the app. So there is no need to add it every class, using it in body is enough